magento-research/pwa-studio
# Remove storybook from peregrine and update references

## Description

Removes storybook from peregrine and all references from the repo.

## Related Issue





Closes #1380.

## Verification Steps

1. Check that I did, in fact, remove storybook from the peregrine repo and that all references to peregrine specific storybook are gone.

## Screenshots / Screen Captures (if appropriate)

## Proposed Labels for Change Type/Package




## Checklist:
